shell/platform/darwin/ios/framework/Source/FlutterOverlayView.h

```diff
@@ -26,13 +28,15 @@
 /// FlutterOverlay view contains the backing stores for the rest. The overlay
 /// views also handle touch propagation and the like for touches that occurs
 /// either on overlays or otherwise may be intercepted by the platform views.
-@interface FlutterOverlayView : UIView
-
+@interface FlutterOverlayView : UIView {
+  fml::CFRef<CGColorSpaceRef> colorSpaceRef;
+}
```

Comment:
In objc ivars that aren't public shoudl be defined in an extension in the .mm file.  They should also have the `_` prefix.

Reply:
Done...?

Reply:
Yep, thanks!